helpers/userHelper.js

```diff
@@ -12,3 +12,15 @@ export const getInitials = (user) => {
     initialsArr.length > 1 ? initialsArr[initialsArr.length - 1].charAt(0) : ''
   }`;
 };
+
+export const getServerToken = () => {
+  const { cookies } = require('next/headers');
```

Comment:
why are you using require and not import at the top of the file?

Reply:
@tomer-tgp , this helper are being users in many places - both sever-side and client-side components.
Because of that, we cannot have that `next/headers` on the top as it's only for server-side.

Reply:
then let's create a separate file - userServerHelper.js and move that specific function there.
the correct imports create the right bundles for us.